src/QtDBMigration.cpp

```diff
@@ -112,6 +126,10 @@ bool QtDBMigrationPrivate::migrate(int destVer)
                            << ", aborting...";
                 return false;
             }
+            else
+            {
+                qWarning() << "Migration successfull!! Version is:" << i;
```

Comment:
Maybe qInfo instead? (or something similar, if none exists, use qDebug)

Reply:
Using qInfo for now.